ecs-init/version/version.go

```diff
@@ -18,10 +18,10 @@
 package version
 
 // Version is the version of the ecs-init
-var Version string = "unknown"
+var Version string = "1.19.0"
```

Comment:
This file should be left untouched as its variables are generated at build time.